torchvision/transforms/functional.py

```diff
@@ -1390,3 +1390,70 @@ def equalize(img: Tensor) -> Tensor:
         return F_pil.equalize(img)
 
     return F_t.equalize(img)
+
+
+def elastic_transform(
+    img: Tensor,
+    alpha: List[float],
+    sigma: List[float],
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    fill: Optional[List[float]] = None,
+) -> Tensor:
```

Comment:
@JLrumberger I wonder if functional op shouldn't be derandomized, it can accept either
1) displacement as [1, H, W, 2] tensor
or
2) same signature as here and we can pass random generator as arg as well (similarly to https://github.com/albumentations-team/albumentations/blob/37e714fd2e326f6f88778e425f98c2de8c8d5372/albumentations/augmentations/geometric/transforms.py#L178)

Reply:
@vfdev-5 this would make it possible to better test the behavior of the op and slightly extend it's functionality. Sounds reasonable to me.

Reply:
I added the requested changes.

Reply:
Thanks for the update, I'll check in details asap. 
Looking at failing tests we may have a limitation from torch jit script side when passing random state. Maybe it is impossible to pass it and thus we have to do something else...
https://app.circleci.com/pipelines/github/pytorch/vision/13819/workflows/452c2c97-199b-47c8-9294-577f77c14583/jobs/1112516